modules/apps/frontend-js/frontend-js-spa-web/package.json

```diff
@@ -2,6 +2,7 @@
 	"dependencies": {
 		"frontend-js-web": "*"
 	},
+	"main": "init.js",
```

Comment:
How does this work? Do we know to look for this inside `src/main/resources/META-INF/resources`? How would webpack know where to find this?

Reply:
This is the key -> https://github.com/liferay/liferay-frontend-projects/blob/master/projects/npm-tools/packages/npm-scripts/src/utils/getProjectMainModuleFilePath.js#L22

Reply:
So yes, we prepend the input directory of npm-scripts